### ISSUE TITLE
Can't use LogglyLogger-CocoaLumberjack with AFNetworking 2.6.0

### ISSUE THREAD

**Issue by ayyoitsp** [melke/LogglyLogger-CocoaLumberjack#20]
Podfile:
```
pod 'AFNetworking', '2.5.4'
pod 'LogglyLogger-CocoaLumberjack', '2.0.3'
```

Results:
```
Analyzing dependencies
[!] Unable to satisfy the following requirements:

- `AFNetworking (= 2.6.0)` required by `Podfile`
- `AFNetworking (~> 2.5.4)` required by `LogglyLogger-CocoaLumberjack/Logger (2.0.3)`
```

**Comment by domness** [comment]
+1

**Comment by melke** [comment]
Fixed in version 2.0.4. AFNetworking up to latest 2.* release is now allowed.
